src/components/ClassCards.scss

```diff
@@ -65,11 +65,12 @@
       &__byline {
         font-size: 0.875rem;
         margin: 0 0 0.5rem;
+        height:  30px;
```

Comment:
I've learned that hard setting height is generally not what you "actually" want to do as if the content wraps to multiple lines we might get into trouble with something being cut off

Reply:
almost always `min-` or `max-` is intended (in my experience)

Reply:
Thanks, I'll bump up the min